include/bluetooth/mesh/light_hue_srv.rst

```diff
@@ -13,7 +13,7 @@ It should be instantiated in the light fixture node.
 The Light Hue Server adds a single model instance to the composition data, in addition to the extended :ref:`bt_mesh_lvl_srv_readme`.
 Although the Light Hue Server contains ``range`` and ``default`` states like other lighting models, the states cannot be accessed directly by other nodes without a :ref:`bt_mesh_light_hsl_srv_readme`.
 
-Unlike the other Lightness models, the Hue Server model's Hue state will wrap around when reaching its maximum value.
+Unlike the other Lightness models, the Hue Server model's Hue state will wrap around when reaching its maximum or minimum values defined in :c:member:`bt_mesh_light_hue_srv.range`.
```

Comment:
```suggestion
Unlike the other Lightness models, the Hue Server model's Hue state will wrap around when reaching its maximum or minimum values as defined in :c:member:`bt_mesh_light_hue_srv.range`.
```
?

Reply:
Is this true though? Should it wrap around when reaching the end of its range like this, or should it only wrap when it's unbound? Seems like it would be a bit odd visually

Reply:
This is how it is stated in the MeshMDLv1.0.1 spec:

Section 6.1.4.1.1 Binding with the Generic Level state:
```
The Light HSL Hue state shall wrap around when reaching the maximum or minimum values.
```

Section 6.1.4.3 Light HSL Hue Range:
```
The Light HSL Hue Range state determines the minimum and maximum hue of an element. 
```

Section 6.1.4.1.3 Binding with the HSL Hue Range state:
```
The Light HSL Hue state is bound to an instance of the Light HSL Hue Range state (see Section 6.1.4.3), meaning
that whenever the Light HSL Hue state of an element changes, the following calculations shall be performed:

Light HSL Hue = Light HSL Hue Range Min
    for values of the Light HSL Hue state that are less than the value of the Light HSL Hue Range Min state
Light HSL Hue = Light HSL Hue Range Max
    for values of the Light HSL Hue state that are greater than the value of the Light HSL Hue Range Max state
```